plugins/azure-devops/src/api/AzureDevOpsClient.ts

```diff
@@ -27,21 +27,28 @@ import {
   RepoBuildOptions,
   Team,
 } from '@backstage/plugin-azure-devops-common';
-import { DiscoveryApi, IdentityApi } from '@backstage/core-plugin-api';
+import {
+  DiscoveryApi,
+  FetchApi,
+  IdentityApi,
+} from '@backstage/core-plugin-api';
 import { ResponseError } from '@backstage/errors';
 import { AzureDevOpsApi } from './AzureDevOpsApi';
 
 /** @public */
 export class AzureDevOpsClient implements AzureDevOpsApi {
   private readonly discoveryApi: DiscoveryApi;
   private readonly identityApi: IdentityApi;
```

Comment:
If this is just a frontend plugin and we're not using this client in the backend, we shouldn't need the `identityApi`, that will get added automatically as part of the `fetchApi` middleware. https://github.com/backstage/backstage/blob/patch/v1.22.0/packages/core-app-api/src/apis/implementations/FetchApi/IdentityAuthInjectorFetchMiddleware.ts#L55-L77

Reply:
Yup, already done ;)